lib/spack/spack/cmd/create.py

```diff
@@ -83,6 +85,22 @@ def install(self, spec, prefix):
         make()
         make('install')"""
 
+    caveats = """\
```

Comment:
Not sure if this is common enough to warrant being in the `spack create` template

Reply:
I'm open to pushback.  If this goes forward and "someone" adds a section to the documentation, then I'd be ok with deleting this.  On the other hand, it needs to be discoverable *somehow* and this seems like a nice start...